.github/workflows/ci_tests.yaml

```diff
@@ -0,0 +1,70 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: PyGMT Tests
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  test:
+    name: ${{ matrix.os }} - Python ${{ matrix.python_version }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      max-parallel: 3
+      fail-fast: false
+      matrix:
+        python_version: [3.7]
+        os: [ubuntu-latest, macOS-latest, windows-latest]
+
+    steps:
+      # Checkout current git repository
+      - name: Checkout
+        uses: actions/checkout@v2.1.0
+
+      # Setup Miniconda
+      - name: Setup Miniconda
+        uses: goanpeca/setup-miniconda@v1.3.1
+        with:
+          python-version: ${{ matrix.python-version }}
+          channels: conda-forge
+
+      # Install GMT and other required dependencies from conda-forge
+      - name: Install GMT and required dependencies
+        shell: bash -l {0}
+        run: conda install gmt=6.0.0 numpy pandas xarray netCDF4 packaging make pytest pytest-cov pytest-mpl sphinx jinja2 docutils ipython
+
+      # Show installed pkg information for postmortem diagnostic
+      - name: List installed packages
+        shell: bash -l {0}
+        run: conda list
+
+      # Cache the ${HOME}/.gmt directory, for docs and testing
+      - name: Cache GMT directory
+        uses: actions/cache@v2
+        id: cache
+        with:
+          path: $(HOME)/.gmt
```

Comment:
GitHub Actions may not support `$(HOME)`. Try `~/.gmt` instead. See the [pip cache example](https://github.com/actions/cache/blob/master/examples.md#python---pip)

Reply:
Ok, won't hurt to try.

Reply:
Now macOS has the correct cache (~6MB), but Linux fails to extract the cache tarballs.

Reply:
Yeah. Not sure why the Windows run is saying it can't find the cache.

Reply:
I'll try using `${{ github.workspace }}/.gmt` instead.

Reply:
Still no luck. I noticed that Linux and Windows agents actually can download some files but timeout for others. For example:
```
Run gmt which -Gu @earth_relief_10m @earth_relief_60m @ridge.txt @Table_5_11.txt @tut_bathy.nc @tut_quakes.ngdc @tut_ship.xyz
earth_relief_10m: Download file from the GMT data server [data set size is 2.8M].
earth_relief_10m: Earth Relief at 10x10 arc minutes obtained by Gaussian Cartesian filtering (18 km fullwidth) of SRTM15+V2 [Tozer et al., 2019].

gmtwhich [ERROR]: Libcurl Error: Timeout was reached
gmtwhich [ERROR]: You can turn remote file download off by setting GMT_DATA_SERVER_LIMIT = 0.
gmtwhich [ERROR]: File earth_relief_10m.grd not found!
earth_relief_60m: Download file from the GMT data server [data set size is 106K].
earth_relief_60m: Earth Relief at 60x60 arc minutes obtained by Gaussian Cartesian filtering (111 km fullwidth) of SRTM15+V2 [Tozer et al., 2019].

C:/Users/runneradmin/.gmt/server/earth_relief_60m.grd
C:/Users/runneradmin/.gmt/cache/ridge.txt
C:/Users/runneradmin/.gmt/cache/Table_5_11.txt
C:/Users/runneradmin/.gmt/cache/tut_bathy.nc
C:/Users/runneradmin/.gmt/cache/tut_quakes.ngdc
C:/Users/runneradmin/.gmt/cache/tut_ship.xyz
```
I think a workaround would be running the "gmt which" command multiple times until all files are downloaded.